src/main/java/org/mule/tools/api/classloader/model/util/DefaultMavenRepositoryLayoutUtils.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Mule ESB Maven Tools
+ * <p>
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * <p>
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.tools.api.classloader.model.util;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.maven.artifact.Artifact;
+import org.apache.maven.artifact.handler.DefaultArtifactHandler;
+
+import java.io.File;
+
+import static java.io.File.separatorChar;
+
+public class DefaultMavenRepositoryLayoutUtils {
+
+  public static String getPomFileName(Artifact artifact) {
+    StringBuilder destFileName = buildMainPOMFileName(artifact);
+
+    destFileName.append("pom");
+
+    return destFileName.toString();
+  }
+
+  public static String getFormattedFileName(Artifact artifact) {
+    StringBuilder destFileName = buildMainFileName(artifact);
```

Comment:
Docs here to please. I understand what you're doing here but given that they are public static methods it looks like "api" stuff